app/backend/src/couchers/models.py

```diff
@@ -181,6 +181,7 @@ class User(Base):
     camping_ok = Column(Boolean, nullable=True)
 
     accepted_tos = Column(Integer, nullable=False, default=0)
+    accepted_community_guidelines = Column(Boolean, nullable=False, server_default="false")
```

Comment:
Could we also make the CG an integer, for same reasons as TOS? Otherwise backend looks fine to me.

Reply:
I thought it was pointless because unlike TOS, we aren't getting it from the backend

Reply:
We need to handle versions of guidelines so make it an int.